aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -39,6 +50,148 @@ fn is_string_type(context: &SafeNativeContext, type_arg: &Type) -> SafeNativeRes
     Ok(false)
 }
 
+/// Given the list of native function arguments, returns a tuple of its
+/// fields: (`aggregator id`, `value`, `limit`).
+pub fn get_aggregator_fields(
```

Comment:
it is a bit awkward to take both arguments together, what if we have a different function with different arguments. i.e. having pair of "pop_value_of_type" and "pop_aggregator_fields_of_type" seems better.

but if you want to keep it as is, you need to make it clear in the comment what it is:

/// Given the list of two arguments (value: T, agg: Aggregator<T>) to a native function, returns a tuple of its fields: (`value`, `aggregator id`, `limit`).

(and I would definitely keep the order of what is returned same as what is passed (i.e. value going before aggregator fields

Reply:
Done.